app/views/shared/_nav_bar.html.erb

```diff
@@ -28,16 +45,12 @@
         <%= drop_down_divider %>
         <%= menu_text "User Account" %>
         <%= menu_item "Manage Account", "#" %>
-        <% if current_user.admin? %>
+        <% if current_user && current_user.admin? %>
           <%= drop_down_divider %>
           <%= menu_text "Admin" %>
           <%= menu_item "Dashboard", "#" %>
         <% end %>
-        <%= drop_down_divider %>
-        <%= menu_item "Logout", logout_path, method: :delete %>
       <% end %>
-    <% else %>
-      <%= menu_item "Login", login_path %>
     <% end %>
   <% end %>
-<% end %>
```

Comment:
Pulled this down to take a look, and it looks awesome!!! :sparkles: Sparkles all around!! :sparkles: